src/App.js

```diff
@@ -1,70 +1,43 @@
-import useList from './hooks/useList'
+import { Switch, Route } from 'react-router-dom'
+import useDatabase from './hooks/useDatabase'
 import styled from 'styled-components/macro'
-import UncheckedList from './02.Components/UncheckedList'
-import CheckedList from './02.Components/CheckedList'
-import AddItemButton from './01.UI-Elements/AddItemButton'
-import UndoButton from './01.UI-Elements/UndoButton'
+//import OverviewPage from './01.Pages/OverviewPage'
```

Comment:
Did you leave the outcomment line on purpose? 🤔

Reply:
i wonder the same thing.  Why is it still there?

Reply:
All was done in preparation for the next user story. But you're right, I should have left it out. THX